src/__tests__/lib/containers/CertificationQuiz.test.tsx

```diff
@@ -0,0 +1,74 @@
+import { render, screen, waitFor } from '@testing-library/react'
+import userEvent from '@testing-library/user-event'
+import React from 'react'
+import CertificationQuiz from '../../../lib/containers/CertificationQuiz'
+import { displayToast } from '../../../lib/containers/ToastMessage'
+import { createWrapper } from '../../../lib/testutils/TestingLibraryUtils'
+import {
+  BackendDestinationEnum,
+  getEndpoint,
+} from '../../../lib/utils/functions/getEndpoint'
+import { Quiz } from '../../../lib/utils/synapseTypes/CertificationQuiz/Quiz'
+import { rest, server } from '../../../mocks/msw/server'
+
+window.open = jest.fn()
+jest.mock('../../../lib/containers/ToastMessage', () => {
+  return { displayToast: jest.fn() }
+})
+
+const mockToastFn = displayToast
+const gettingStartedUrl =
+  'https://help.synapse.org/docs/Getting-Started.2055471150.html'
+
+const getQuizHandler = rest.get(
+  `${getEndpoint(
+    BackendDestinationEnum.REPO_ENDPOINT,
+  )}/repo/v1/certifiedUserTest`,
+  async (req, res, ctx) => {
+    const response: Quiz = {
+      id: 123,
+      header: 'Mock Certification Quiz',
+      questions: [],
```

Comment:
Would like to see 1-2 mock questions in here. Would want to see these additional scenarios: 
- Choose an answer, verify that the selected answer was submitted
- Simulate case where submit clicked and not all questions have been answered
- Simulate case where all answers are correct
- Simulate case where an answer was wrong